src/ncp/ncp_openthread.cpp

```diff
@@ -285,6 +286,9 @@ otError ControllerOpenThread::ApplyFeatureFlagList(const FeatureFlagList &aFeatu
 #if OTBR_ENABLE_DHCP6_PD
     otBorderRoutingDhcp6PdSetEnabled(mInstance, aFeatureFlagList.enable_dhcp6_pd());
 #endif
+#if OTBR_ENABLE_LINK_METRICS_TELEMETRY
```

Comment:
`_TELEMETRY` or `_MANAGER`?

Reply:
In ot-br-posix, the feature flag is `OTBR_LINK_METRICS_TELEMETRY`. In the `CMakeLists.txt`(https://github.com/openthread/ot-br-posix/blob/main/third_party/openthread/CMakeLists.txt#L55), when this is ON, it will set the corresponding feature option in openthread:
```
set(OT_LINK_METRICS_INITIATOR ${OTBR_LINK_METRICS_TELEMETRY} CACHE STRING "enable link metrics initiator" FORCE)
set(OT_LINK_METRICS_MANAGER ${OTBR_LINK_METRICS_TELEMETRY} CACHE STRING "enable link metrics manager" FORCE)
```